tests/Humbug/Test/SelfUpdate/UpdaterGithubStrategyTest.php

```diff
@@ -99,6 +99,12 @@ public function testSetStabilityThrowsExceptionOnInvalidStabilityValue()
      */
     public function testUpdatePhar()
     {
+        if (!extension_loaded('openssl')) {
+            $this->markTestSkipped('This test requires the openssl extension to run.');
+
+            return;
```

Comment:
no need to return here (``markTestSkipped`` actually throw an exception to stop the test)